packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -313,7 +313,15 @@ export default abstract class BaseCall implements IWebRTCCall {
     }
   }
 
+  /**
+   * Transfer a call.
+   *
+   * Note: This method is work-in-progress and not currently working.
```

Comment:
I would hide this from react-developer-docs:
```suggestion
   * @ignore Note: This method is work-in-progress and not currently working.
```